test-base/src/main/java/kin/ecosystem/test/base/ClientModifierFake.kt

```diff
@@ -0,0 +1,33 @@
+package kin.ecosystem.test.base
+
+import com.kin.ecosystem.core.bi.EventsStore.ClientModifier
+import com.kin.ecosystem.core.bi.EventsStore.DynamicValue
+import com.kin.ecosystem.core.bi.events.ClientProxy
+
+class ClientModifierFake : ClientModifier {
+
+    private var os: String? = null
```

Comment:
maybe use `lateinit` instead of assigning `null`s?
or maybe you don't even need to because you're using the `init` block?

Reply:
it was auto conversion from Java to Kotlin by the IDE